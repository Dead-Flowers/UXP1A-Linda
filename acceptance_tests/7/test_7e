-t 20 input '(integer:1)'
