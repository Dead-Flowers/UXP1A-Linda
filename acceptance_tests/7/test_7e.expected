(1)
Exit code: 0
